src/webgpu/api/validation/image_copy/texture_related.spec.ts

```diff
@@ -89,31 +89,38 @@ g.test('usage')
   });
 
 g.test('sample_count')
-  .desc(`Multisampled textures cannot be copied.`)
+  .desc(`It must be a full copy if the texture is multisampled.`)
   .params(u =>
     u //
       .combine('method', kImageCopyTypes)
```

Comment:
Copying multisampled textures is only supported for copyT2T, not copyB2T and copyT2B

Reply:
The validation tests on T2T multisampled texture copies are here:
https://github.com/gpuweb/cts/blob/main/src/webgpu/api/validation/encoding/cmds/copyTextureToTexture.spec.ts

The operation tests on T2T multisampled texture copies haven't been added yet. Tracking Dawn issue: https://bugs.chromium.org/p/dawn/issues/detail?id=1033